aten/src/ATen/InferSize.h

```diff
@@ -41,12 +46,24 @@ inline std::vector<int64_t> infer_size(IntArrayRef shape, int64_t numel) {
                "value and is ambiguous");
       res[*infer_dim] = numel / newsize;
     }
-    return res;
+    return;
```

Comment:
nit: no need to return at the end of a void function

Reply:
true, but this isn't at the end :)